fews_3di/simulation.py

```diff
@@ -93,12 +92,13 @@ class ThreediSimulation:
     simulation_url: str
     simulations_api: openapi_client.SimulationsApi
     threedimodels_api: openapi_client.ThreedimodelsApi
-
+    
     def __init__(
         self, settings: utils.Settings, allow_missing_saved_state: bool = False
     ):
         """Set up a 3di API connection."""
         self.settings = settings
+        API_HOST = self.settings.api_host
```

Comment:
`ALL_CAPS` indicates a constant, which this is not.

I'd suggest just leaving out the entire line. Just use `self.settings.api_host` where you need it. On those lines, you also use `self.settings.username`, so that fits together.